.github/workflows/harmony-in-a-box.yml

```diff
@@ -5,7 +5,7 @@ on: [push]
 jobs:
   harmony-in-a-box:
 
-    runs-on: macos-latest
+    runs-on: macos-12
```

Comment:
Maybe add a TODO comment with a ticket number explaining why it is locked at 12 and that we want to use latest again.

Reply:
Done.